ui/articles-aem-renderer/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
the root `.gitignore` already takes care of this, so there isn't a need to include the additional `.gitignore` file

Reply:
The wizard added it I removed it.